src/auth-service/models/Maintenance.js

```
@@ -0,0 +1,248 @@
+      logObject("error in the object", err);
+      logger.error(`Data conflicts detected -- ${err.message}`);
+      let response = {};
+      let errors = {};
+      let message = "Internal Server Error";
+      let status = httpStatus.INTERNAL_SERVER_ERROR;
+      if (err.code === 11000 || err.code === 11001) {
+        errors = err.keyValue;
+        message = "duplicate values provided";
+        status = httpStatus.CONFLICT;
+        Object.entries(errors).forEach(([key, value]) => {
+          return (response[key] = value);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)